src/main/flight/pid.h

```diff
@@ -23,8 +23,11 @@
 #include <stdbool.h>
 #include "common/time.h"
 #include "common/filter.h"
+#include "common/axis.h"
 #include "pg/pg.h"
 
+#include "sensors/acceleration.h"
```

Comment:
This looks good now, but I'd actually move this `#include` down into the `#ifdef UNIT_TEST` block, to make it explicit that this is only required for the unit tests.

Reply:
Good point indeed